python/the-basics/initial-setup.html.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Setting up a Python Environment"
```

Comment:
Just a heads up, it was a bit confusing for me to have this page (and the `Multi-stage Builds` page) as a link. It wasn't intuitive that those were other clickable pages.

Reply:
It became more clear when I accessed other main pages.

Reply:
that's a fair point; maybe we can bring this up as a design thing? The same issue will apply to js and rust docs too as they follow the same pattern

Reply:
Yes, they can suggest some changes/improvements in general 😁